libsolidity/interface/Natspec.cpp

```diff
@@ -110,9 +127,24 @@ Json::Value Natspec::devDocumentation(ContractDefinition const& _contractDef)
 				methods[it.second->externalSignature()] = method;
 			}
 		}
+		if (auto var = dynamic_cast<VariableDeclaration const*>(&it.second->declaration()))
+		{
+			Json::Value method(devDocumentation(var->annotation().docTags));
+			if (!method.empty() && var->isStateVariable() && var->isPublic())
+				methods[it.second->externalSignature()] = method;
+		}
+	}
+
+	for (auto const& it: _contractDef.stateVariables())
+	{
+		Json::Value documentation(devDocumentation(it->annotation().docTags));
+		if (!documentation.empty() && it->visibility() != Visibility::Public)
```

Comment:
I'm not sure if we should not just put it here always.

Reply:
thats true.. i think always would probably make it easier for some tool devs.. I will put it always